src/pages/submit/poll.tsx

```diff
@@ -129,7 +129,7 @@ export default function SubmitPoll() {
         })
         .then((proposal) => {
           loader.proposals.set(proposal.id, proposal)
-          navigate(locations.proposal(proposal.id), { replace: true })
+          navigate(locations.proposal(proposal.id, true), { replace: true })
```

Comment:
use the options and shorter version

```suggestion
            navigate(locations.proposal(proposal.id,{ new: true }), { replace: true })
```